README.md

```diff
@@ -6,24 +6,93 @@
 
 flannel is a virtual network that gives a subnet to each host for use with container runtimes.
 
-Platforms like Kubernetes assume that each container (pod) has a unique, routable IP inside the cluster.
-The advantage of this model is that it reduces the complexity of doing port mapping.
+Platforms like Kubernetes assume that each container (pod) has a unique, routable IP inside the cluster. The advantage of this model is that it reduces the complexity of doing port mapping.
 
 ## Theory of Operation
 
-flannel runs an agent, flanneld, on each host and is responsible for allocating a subnet lease out of a preconfigured address space.
-flannel uses [etcd](https://github.com/coreos/etcd) to store the network configuration, allocated subnets, and auxiliary data (such as host's IP).
-The forwarding of packets is achieved using one of several strategies that are known as backends.
-The simplest backend is `udp` and uses a TUN device to encapsulate every IP fragment in a UDP packet, forming an overlay network.
+Flannel runs an agent, `flanneld`, on each host and is responsible for allocating a subnet lease out of a preconfigured address space. Flannel uses [etcd][etcd] to store the network configuration, allocated subnets, and auxiliary data (such as host's IP). The forwarding of packets is achieved using one of several strategies that are known as [backends][backends].
+
 The following diagram demonstrates the path a packet takes as it traverses the overlay network:
 
 ![Life of a packet](./packet-01.png)
 
-## Documenation
-- [Running](Documentation/running.md)
+## Getting started
+
+It's really easy to use flannel with Kubernetes. Flannel supports using the Kubernetes API server as its backing datastore which means there's no need to deploy an etcd server for flannel, or even for flannel to access the etcd server that Kubernetes uses. This flannel mode is known as the *kube subnet manager* and is demonstrated below.
+
+### Adding flannel
+
+Flannel can be added to any existing Kubernetres cluster. It's easiest to add flannel before any pods have been started (host networked pods are fine).
```

Comment:
s/Kubernetres/Kubernetes/
second sentence: "It's simplest to add flannel before any pods using the pod network have been started."